Makefile

```diff
@@ -65,13 +66,13 @@ docker_build_master:
 
 .PHONY: docker_push_release
```

Comment:
can we change the name of these commands?.

Reply:
Its possible. I would not see `docker_push_release` referring to the registry but rather the way in which we push the image
/2cents